### ISSUE TITLE
Spurious deprecation warnings

### ISSUE THREAD

**Issue by wchargin** [tensorflow/tensorflow#27897]
**System information**

- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): Yes
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): gLinux
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device: N/A
- TensorFlow installed from (source or binary): tf-nightly-2.0-preview-20190416-py2.7
- TensorFlow version (use command below): 2.0.0-dev20190416
- Python version: 2.7.16rc1
- Bazel version (if compiling from source): N/A
- GCC/Compiler version (if compiling from source): N/A
- CUDA/cuDNN version: N/A
- GPU model and memory: N/A

**Describe the current behavior**

When using `keras.callbacks.TensorBoard` or `keras.models.Sequential`, a
deprecation warning is emitted instructing the user to use the same
symbol that they’re already using:

>   - “The name keras.callbacks.TensorBoard is deprecated. Please use keras.callbacks.TensorBoard instead.”
>   - “The name keras.models.Sequential is deprecated. Please use keras.models.Sequential instead.”

**Describe the expected behavior**

No deprecation warning should be emitted.

**Code to reproduce the issue**

```
$ cat /tmp/repro.py; echo
import tensorflow as tf
print(tf.__version__)
tf.keras.callbacks.TensorBoard
tf.keras.models.Sequential

$ python /tmp/repro.py
2.0.0-dev20190416
WARNING: Logging before flag parsing goes to stderr.
W0416 11:29:00.042920 139891507525376 deprecation_wrapper.py:76] From /tmp/repro.py:3: The name keras.callbacks.TensorBoard is deprecated. Please use keras.callbacks.TensorBoard instead.

W0416 11:29:00.043119 139891507525376 deprecation_wrapper.py:76] From /tmp/repro.py:4: The name keras.models.Sequential is deprecated. Please use keras.models.Sequential instead.

```

**Comment by wchargin** [comment]
Looks like this happens with lots of symbols:

```
$ cat /tmp/repro.py; echo
import tensorflow as tf
print(tf.__version__)
tf.io.gfile.exists
tf.errors.OpError
tf.train.Example

$ python /tmp/repro.py
2.0.0-dev20190416
WARNING: Logging before flag parsing goes to stderr.
W0416 12:01:54.716809 140711866500864 deprecation_wrapper.py:76] From /tmp/repro.py:3: The name io.gfile.exists is deprecated. Please use io.gfile.exists instead.

W0416 12:01:54.716949 140711866500864 deprecation_wrapper.py:76] From /tmp/repro.py:4: The name errors.OpError is deprecated. Please use errors.OpError instead.

W0416 12:01:54.716995 140711866500864 deprecation_wrapper.py:76] From /tmp/repro.py:5: The name train.Example is deprecated. Please use train.Example instead.

```

**Comment by mihaimaruseac** [comment]
Makes sense: the warnings are there to inform you that on the next version of TensorFlow those symbols will no longer be present, so you should update your code to the new API.

See also: #27045, #27023, #27016, #26844, #26151, #26149, #26144, and many other similar issues.

**Comment by wchargin** [comment]
Hmm, as far as I can tell I _am_ using the new APIs, and the warnings
are telling me to migrate to the same APIs that I’m already using:

> “The name **keras.models.Sequential** is deprecated. Please use
> **keras.models.Sequential** instead.”

How should I change “`tf.keras.models.Sequential`” to avoid a warning?

**Comment by mihaimaruseac** [comment]
Oh, my bad, I didn't notice that. Sorry

**Comment by wchargin** [comment]
No worries; thanks.

**Comment by annarev** [comment]
Thank you for catching and sorry for the trouble! Fix is on the way.

**Comment by annarev** [comment]
Should be fixed by this commit: e84d155152ab5a187d2c0e4c5eea6a73df3789e1 
Let me know if you are still seeing issues!
